src/librustc_codegen_ssa/mir/block.rs

```diff
@@ -34,20 +34,20 @@ struct TerminatorCodegenHelper<'a, 'tcx> {
 impl<'a, 'tcx> TerminatorCodegenHelper<'a, 'tcx> {
     /// Returns the associated funclet from `FunctionCx::funclets` for the
     /// `funclet_bb` member if it is not `None`.
-    fn funclet<'c, 'b, Bx: BuilderMethods<'b, 'tcx>>(
+    fn funclet<'d, 'c, 'b, Bx: BuilderMethods<'b, 'tcx>>(
```

Comment:
The ordering of lifetimes didn't seem to follow a pattern. Since I'm already modifying these lines, should I go ahead and make them all nice and consistent?